src/Engine/Manager/ScriptManager.cpp

```diff
@@ -782,12 +817,6 @@ int ScriptManager::GetStringDeclarationID() {
 
 const int ScriptManager::GetSizeOfASType(int typeID, void* value) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetSizeOfASType' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)